### ISSUE TITLE
[A11Y] Unable to exit overflow menu

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3894]
When a11y user is inside the overflow menu than currently there is no easy way to exit it. This might be a problem with the screen-reader (Talkback) itself as I have tested this on other apps like Whatsapp, Instagram and the experience is equally bad.

https://user-images.githubusercontent.com/9396084/136244259-9c79bc6a-ca2a-477f-9ec4-fec05447e99a.mp4

**Comment by rt4914** [comment]
We might need to use `BottomSheet` instead of `Overflow Menu`. Need to discuss with design team.
